index.html

```diff
@@ -2,32 +2,32 @@
 <html lang="en">
 
 <head>
-	<meta charset="utf-8">
-	<title>Orange Inc.</title>
-	<link rel="stylesheet" href="css/styles.css">
-	<script src="js/script.js"></script>
+  <meta charset="utf-8">
+  <title>Orange Inc.</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="js/script.js"></script>
 </head>
 
 <body>
 
-<div class="header">
-	<h1>-todos-</h1>
-	<input id="textbox" type="text" class="inputtextbox">
-	<input id="" class="submitbutton" type="submit" value="Submit" onclick="addItem()">
-</div>	
+  <div class="header">
+    <h1>-todos-</h1>
+    <input id="textbox" type="text" class="inputtextbox">
+    <input id="" class="submitbutton" type="submit" value="Submit" onclick="addItem()">
+  </div>
 
-<div class="listdiv">
-	<ul id="list">
-	</ul>
-</div>
+  <div class="listdiv">
```

Comment:
bad class name. You should never use `div` in the class name. Be more descriptive, something like: 
`todo-list-container` or `todo-list-wrapper`